### PR TITLE
Testing/donation form modal cr

### PR DESCRIPTION
**Pull request by Fordo29**
### 1. What changed?
We added testing for the donation page to ensure all elements are displayed on the page. 

### 2. Why is this change necessary?
We needed to test the elements on the page

### 3. What changed technically that may impact others?
We added tests which we need to ensure passes for Github Actions CI

### 4. How do we test it?
Please see the Github Actions CI in this repo.

### CODE REVIEW

File: cypress/integration/donation_form_spec.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+describe('Donation Form Modal', () => {
```
Comment (anewb87) [inline cypress/integration/donation_form_spec.js:1]: We have officially written tests for all the elements on the form/modal.

**Review by anewb87 — APPROVED** [review]
Going forward we're working to create the habit of writing and updating tests as we write JS rather than at the very end of the project.

File: cypress/integration/donation_form_spec.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      .contains('Age (years)')
+      .get('label')
+      .contains('Type')
+      .get('label')
+      .contains('Gender')
+      .get('label')
+      .contains('Description')
+      .get('label')
+      .contains('Add an image of your pet:')
+      .get('[data-testid="image-upload"]')
```
Comment (ectweitmann) [inline cypress/integration/donation_form_spec.js:28]: This was cool to learn! Nice job Christine!

**Review by ectweitmann — APPROVED** [review]

File: cypress/integration/donation_form_spec.js (hunk 2 above)
Comment (StevieAmb) [inline cypress/integration/donation_form_spec.js:28]: Great work using this "data-testid"!

**Review by StevieAmb — APPROVED** [review]

**Pull request merged**